packages/wonder-blocks-core/util/add-style.js

```diff
@@ -29,3 +33,10 @@ export default function addStyle<T: Object>(
 
     return MediaLayoutWrapper(StyleComponent);
 }
+
+// TODO(kevinb): add more style resets
+const styles = StyleSheet.create({
+    button: {
+        margin: 0, // Safari adds 2px left/right margins
+    },
```

Comment:
@wishdasher we could add other resets based on normalizer.css here if necessary.

Reply:
🤷‍♂️ I'm... iffy on this solution. I could see how this would be useful but since we're applying it intermittently I could also see how it'd be really confusing. I think this is ok for now but I think we should either flush this out a lot more or move it to be in the components themselves.

I think it'd be good to add a comment explaining the way in which we intend this to be used, so that we don't add in non-browser-specific overrides (also probably improve the variable name so that it's clearer, too).

Reply:
I'll add some comments and improve the naming so that people don't abuse this.